games-mud/tkmoo/tkmoo-0.3.32-r1.ebuild

```diff
@@ -1,44 +1,48 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit eutils
+EAPI=8
 
-MY_PN=${PN/moo/MOO-light}
-MY_P=${P/moo/MOO-light}
+inherit desktop
+
+MY_PN="${PN/moo/MOO-light}"
+MY_P="${P/moo/MOO-light}"
 DESCRIPTION="MOO Client written in Tcl/Tk"
```

Comment:
Tiny nitpick but I prefer a blank line after variables before description.

Also, I find the substitution kind of silly, and done twice for nothing, e.g. /could/
```
MY_PN="tkMOO-light"
MY_P="${MY_PN}-${PV}"
```

However see my next comment, I think `MY_P="tkMOO-light-${PV}"` is all we need here.